README.md

```diff
@@ -13,6 +14,8 @@
 
 Full documentation here: https://lf1-io.github.io/padl/
 
+Try out our example notebooks here: https://colab.research.google.com/github/lf1-io/padl/
```

Comment:
@jasonkhadka Is this alright?

Reply:
i have updated the README with direct listing of colab notebooks. I think that is clearer.